### DISCUSSION TITLE
Use PlanetScale Serverless Driver for Prisma Database Connection

### DISCUSSION THREAD

**Discussion by kirkouimet** [prisma/prisma#15553] [Q&A] [answered]
We are using Cloudflare Workers with the PlanetScale Serverless Driver:

https://planetscale.com/blog/introducing-the-planetscale-serverless-driver-for-javascript

We would like to adopt Prisma as our ORM. Prisma currently requires a direct MySQL connection, which Cloudflare Workers do not support. The alternative is to use Prisma's Data Proxy, but we would like to avoid having to use this as PlanetScale's Serverless Driver already handles connection pooling.

Is it possible to create a custom connection in Prisma which uses PlanetScale's Serverless Driver?

Thanks!

**Comment by janpio** [comment, marked as answer]
There is an issue tracking this feature request here: https://github.com/prisma/prisma/issues/15265

Right now there are not plans to jump on this, as it unfortunately is not trivial at all from an ORM perspective.

**Reply by kirkouimet** [reply]
Thank you @janpio!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
